### ISSUE TITLE
Meshing and texture mapping a .ply file using ReconstructMesh and TextureMesh executable

### ISSUE THREAD

**Issue by kirangani** [cdcseacave/openMVS#339]

**Comment by varunkhare1234** [comment]
check issue #225 I am also trying to get it work there @slice-of-life

**Comment by slice-of-life** [comment]
Have you tried using Reconstructmesh executable to create a mesh on .ply file if we know associated views?  @varunkhare1234

**Comment by varunkhare1234** [comment]
No, I haven't tried reconstruct mesh. I just wanted to apply a texture to my .ply file and finally used meshlab for it.

**Comment by cdcseacave** [comment]
Closing as stale. The OpenMVS codebase has undergone extensive refactoring and modernisation on the \develop\ branch since this issue was opened. If you believe this is still relevant, please test against the latest \develop\ branch and open a new issue with reproduction steps.
